httpcache_test.go

```diff
@@ -102,29 +114,7 @@ func TestExistingAgeHeader(t *testing.T) {
 	}`, "caddyfile")
 
 	resp1, _ := tester.AssertGetResponse(`http://localhost:9080/age-header`, 200, "Hello, Age header!")
-	age, _ := strconv.Atoi(resp1.Header.Get("Age"))
-	if age <= 0 {
+	if resp1.Header.Get("Age") != "" {
```

Comment:
We should keep the previous test which is more accurate.

Reply:
On the first response we must not set the Age header. The Age header must be set when the response is served from the cache. https://datatracker.ietf.org/doc/html/rfc7234#section-4.2.3

Reply:
I don't see where it's stated that the header must not be set when not coming from the cache and IIRC Varnish always set it.

Anyway the name of this test is `TestExistingAgeHeader`, so it what it must do (trigger a second request if necessary).

Reply:
I'll add the second request.